src/index.js

```diff
@@ -1,13 +1,53 @@
+import _ from 'lodash';
+
+// TODO setup all mongodb indexes
+
 class Mongo {
```

Comment:
This class should implement the flow interface DBDriver right? I'm not quite sure the syntax that flow requires for this. @davidyaha Thoughts?